static/app/views/settings/project/loaderScript.tsx

```diff
@@ -0,0 +1,141 @@
+              return index === existingProjectIndex ? projectKey : updatedProjectKey;
+            })
+          : [...updatedProjectKeys, projectKey];
+
+      setUpdatedProjectKeys(newUpdatedProjectKeys);
+    },
+    [updatedProjectKeys]
+  );
+
+  return (
+    <div>
+      <SettingsPageHeader title={t('Loader Script')} />
+
+      <TextBlock>
+        {tct(
+          'The Loader Script is the easiest way to initialize the Sentry SDK. The Loader Script automatically keeps your Sentry SDK up to date and offers configuration for different Sentry features. [docsLink:Learn more about the Loader Script]. Note: The Loader Script is bound to a Client Key (DSN), to create a new Script, go to the [clientKeysLink:Client Keys page].',
+          {
+            docsLink: (
+              <ExternalLink href="https://docs.sentry.io/platforms/javascript/install/loader/" />
+            ),
+            clientKeysLink: (
+              <Link
+                to={`/settings/${organization.slug}/projects/${project.slug}/keys/`}
+              />
+            ),
+          }
+        )}
+      </TextBlock>
+
+      {isLoading && <LoadingIndicator />}
+      {!!error && (
+        <LoadingError
+          message={t('Failed to load project keys.')}
+          onRetry={refetchProjectKeys}
+        />
+      )}
+      {!isLoading && !error && !projectKeys?.length && (
+        <EmptyMessage title={t('There are no keys active for this project.')} />
+      )}
+
+      {projectKeys?.map(key => {
+        const actualKey =
+          updatedProjectKeys.find(updatedKey => updatedKey.id === key.id) ?? key;
+        return (
+          <LoaderItem
+            key={actualKey.id}
+            organization={organization}
+            project={project}
+            projectKey={actualKey}
+            updateProjectKey={handleUpdateProjectKey}
+          />
+        );
+      })}
+    </div>
+  );
+}
+
+function LoaderItem({
+  organization,
+  project,
+  projectKey,
+  updateProjectKey,
+}: {
+  organization: Organization;
+  project: Project;
+  projectKey: ProjectKey;
+  updateProjectKey: (projectKey: ProjectKey) => void;
+}) {
+  return (
+    <Panel>
+      <PanelHeader hasButtons>
+        <div>{tct('Client Key: [name]', {name: projectKey.name})}</div>
+        <LinkButton
+          to={`/settings/${organization.slug}/projects/${project.slug}/keys/${projectKey.id}/`}
+        >
+          {t('View Key Details')}
+        </LinkButton>
+      </PanelHeader>
+      <PanelBody>
+        <PanelAlert type="info" showIcon>
+          {t('Note that it can take a few minutes until changed options are live.')}
+        </PanelAlert>
+
+        <LoaderSettings
+          orgSlug={organization.slug}
+          keyId={projectKey.id}
+          project={project}
+          data={projectKey}
+          updateData={updateProjectKey}
+        />
+      </PanelBody>
+    </Panel>
+  );
+}
+
+export default withOrganization(ProjectLoaderScript);
```

Comment:
instead of `withOrganization` shouldn't we use the hook `useOrganization`?

Reply:
nit: named export too :)

Reply:
done!